tests/PriorityMapTest.php

```diff
@@ -89,22 +91,33 @@ public function testGetKeys()
         $list->set('k4', 'v4', 1);
         $list->set('k5', 'v5', 4);
 
-        $this->assertEquals(
-            array('k1', 'k2', 'k3', 'k4', 'k5'),
-            $list->getKeys()
-        );
+        $result = $list->getKeys();
+
+        $this->assertCount(5, $result);
+        foreach($result as $key) {
+            $this->assertContains($key, $result);
+        }
+    }
+
+    public function testCount()
+    {
+        $list = new PriorityMap();
+
+        $list->set('K1', 'v1', 2);
+
+        $this->assertSame(1, $list->count());
```

Comment:
your test must check border conditions: what result returns method `count` for empty priority list

Reply:
Ok, I create the another method named ```testCountWithEmptyList```.